.travis.yml

```diff
@@ -104,6 +104,13 @@ jobs:
         #   launching minikube test environment
         - travis_terminate 0    # deploy only on x86
 
+    - stage: deploy testing
+      name: run deploy section as test
+      script:
+        - ./deploy.sh || travis_terminate 1
```

Comment:
@nixpanic will this push any image or helm charts? will this get tested as the this is pull request https://github.com/ceph/ceph-csi/blob/7d642b791c1bdae966dbc5338ea6efd4200dc07c/deploy.sh#L82

Reply:
oh, maybe it does nothing at all. Restructuring the script and sharing the functions with the multi-arch build too now, stay tuned for an update :wink: